server/routes/routerRequestHandlers.test.ts

```diff
@@ -397,10 +396,8 @@ describe('routerRequestHandlers', () => {
       const prisonNumber = 'A1234GC'
       const prisonId = 'MDI'
       req.params.prisonNumber = prisonNumber
```

Comment:
See my comments below on `PrisonerSearchService` to understand why this test and its implementation have changed